package.json

```diff
@@ -19,25 +19,29 @@
     "@types/node": "^12.12.14",
     "@types/node-fetch": "^2.1.2",
     "@types/progress": "2.0.3",
+    "@typescript-eslint/eslint-plugin": "4.24.0",
+    "@typescript-eslint/parser": "4.24.0",
     "durations": "3.4.2",
+    "eslint": "7.26.0",
+    "eslint-config-mural": "1.1.0",
+    "eslint-config-prettier": "8.3.0",
+    "eslint-plugin-muralco": "1.1.4",
+    "eslint-plugin-prettier": "3.4.0",
     "execa": "5.0.0",
+    "prettier": "2.0.5",
     "ts-unused-exports": "^7.0.3",
-    "tslint": "^5.11.0",
-    "tslint-config-airbnb": "^5.9.2",
     "typescript": "^3.7.3"
   },
   "dependencies": {
     "@cucumber/cucumber": "7.2.1",
     "node-fetch": "^2.2.0",
     "ts-node": "^7.0.0"
   },
-  "peerDependencies": {
```

Comment:
I think we shouldn't delete this from peer dependencies since this module works on top of it we can make the user of this module decide which version of cucumber to use

Reply:
Actually thinks get broken if you install your own version of cucumber, I added that.

Reply:
Once removed/improved some monkeypatches in `src/debug/index.ts` we should add this back. For now is ok